### ISSUE TITLE
Cloud Logging always enabled when Cloud Monitoring is enabled

### ISSUE THREAD

**Issue by sok1234** [terraform-google-modules/terraform-google-kubernetes-engine#1321]
### TL;DR

I'm using beta-private-cluster-update-variant module version 22.0.0 and I can't find a way to only enable Cloud Monitoring.

I can either disable both or enable both.

I've tried all possible combinations of logging_service/monitoring_service and logging_enabled_components/monitoring_enabled_components but none seems to work.

### Expected behavior

Using the right values for logging/monitoring I would like to have:

Cloud Logging: Disabled
Cloud Monitoring: Enabled

### Observed behavior

I can only configure it in two ways:

Both Cloud Logging/Monitoring Disabled
Both Cloud Logging/Monitoring Enabled

### Terraform Configuration

```hcl
module "gke" {
  source                         = "terraform-google-modules/kubernetes-engine/google//modules/beta-private-cluster-update-variant"
  version                        = "22.0.0"
  ...
  monitoring_enabled_components  = ["SYSTEM_COMPONENTS", "WORKLOADS"]
  logging_service                = "none"
  ...
}
```

### Terraform Version

```sh
$ terraform version
Terraform v1.1.0
on linux_amd64
```
